doc/configcommands.dsv

```diff
@@ -8,7 +8,7 @@ bind-key||<key> <operation> [<dialog>]||n/a||Bind key <key> to <operation>. This
 bookmark-autopilot||[yes/no]||no||If set to `yes`, the configured bookmark command is executed without any further input asked from user, unless the url or the title cannot be found/guessed.||bookmark-autopilot yes
 bookmark-cmd||<command>||""||If set, then <command> will be used as bookmarking plugin. See the documentation on bookmarking for further information.||bookmark-cmd "~/bin/delicious-bookmark.sh"
 bookmark-interactive||[yes/no]||no||If set to `yes`, then the configured bookmark command is an interactive program.||bookmark-interactive yes
-browser||<command>||lynx||Set the browser command to use when opening an article in the browser. If <command> contains `%u`, it will be used as complete commandline and `%u` will be replaced with the URL that shall be opened.||browser "w3m %u"
+browser||<command>||lynx||Set the browser command to use when opening an article in the browser. If BROWSER environment variable is set, it will be used as the default browser. If <command> contains `%u`, it will be used as complete commandline and `%u` will be replaced with the URL that shall be opened.||browser "w3m %u"
```

Comment:
I think third column (one that says "lynx") should be updated too. We don't have a precedent for the format; I suggest "$BROWSER, otherwise lynx".

I'd also add "otherwise lynx will be used" to the description you already provided, to make the precedence crystal clear.